app/models/review.rb

```diff
@@ -0,0 +1,13 @@
+class Review < ApplicationRecord
+  belongs_to :art
+  belongs_to :user
+
+  validates :rating, presence: true, inclusion: 1..5, numericality: { only_integer: true }
+  validate :user_has_booked_before
+
+  private
+
+  def user_had_booked_before
+    user.bookings.map(&:art).include(art)
+  end
```

Comment:
Btw this is not correct.
I just made a quick example in slack to show my idea but that's not how you should write it.
Try taking example of other custom validations we have.

include inst' the correct method either, it should be include?.

Don't forget to always try your validations in the console to see if everything works